src/export_data.py

```diff
@@ -0,0 +1,145 @@
+"""Copyright 2020 Google LLC.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+# Can be run as follows:
+# python3 export_data.py --params_path=candid.json --export_folder=samples
+
+import json
+import argparse
+import ee
+from utils import TaskManager, SampleExporter, TropomiImagery, DSMImagery
+from utils import RoadImagery, MultiSpectralImagery, WindImagery, stack_bands_from_imagery
+
+if __name__ == "__main__":
```

Comment:
This is a long __main__ that does a lot of things. I recommend separating responsibilities in here into methods so that arg parsing, config file parsing, EE setup, and the main loop aren't all crammed into the same place.